mapie/regression/utils/ccp_phi_function.py

```diff
@@ -0,0 +1,974 @@
+from __future__ import annotations
+
+from abc import ABCMeta, abstractmethod
+import inspect
+from typing import (Iterable, Callable, Optional, Tuple, Union,
+                    cast, Dict, List, Any)
+import numbers
+import warnings
+
+import numpy as np
+from mapie._typing import ArrayLike, NDArray
+from sklearn.utils import _safe_indexing
+from sklearn.utils.validation import _num_samples, check_is_fitted
+from sklearn.base import BaseEstimator
+
+
+def _is_fitted(
+    estimator: Any,
+    attributes: Optional[Union[List[str], str]] = None,
+    all_or_any: Callable = all,
+):
```

Comment:
Two choices: move to utils.py or remove it because duplication function.

Reply:
I removed it, by using fit_attributes and check_is_fitted from sklearn